plugins/modules/rds_cluster.py

```diff
@@ -0,0 +1,1024 @@
+        type: bool
+        default: True
+    wait:
+        description: Whether to wait for the cluster to be available or deleted.
+        type: bool
+        default: True
+    # Options that have a corresponding boto3 parameter
+    apply_immediately:
+        description:
+          - A value that specifies whether modifying a cluster with I(new_db_cluster_identifier) and I(master_user_password)
+            should be applied as soon as possible, regardless of the I(preferred_maintenance_window) setting. If C(False), changes
+            are applied during the next maintenance window.
+        type: bool
+        default: False
+    availability_zones:
+        description:
+          - A list of EC2 Availability Zones that instances in the DB cluster can be created in.
+            May be used when creating a cluster or when restoring from S3 or a snapshot.
+        aliases:
+          - zones
+          - az
+        type: list
+        elements: str
+    backtrack_to:
+        description:
+          - The timestamp of the time to backtrack the DB cluster to in ISO 8601 format, such as "2017-07-08T18:00Z".
+        type: str
+    backtrack_window:
+        description:
+          - The target backtrack window, in seconds. To disable backtracking, set this value to C(0).
+          - If specified, this value must be set to a number from C(0) to C(259,200) (72 hours).
+        default: 0
+        type: int
+    backup_retention_period:
+        description:
+          - The number of days for which automated backups are retained (must be within C(1) to C(35)).
+            May be used when creating a new cluster, when restoring from S3, or when modifying a cluster.
+        type: int
+        default: 1
+    character_set_name:
+        description:
+          - The character set to associate with the DB cluster.
+        type: str
+    database_name:
+        description:
+          - The name for your database. If a name is not provided Amazon RDS will not create a database.
+        aliases:
+          - db_name
+        type: str
+    db_cluster_identifier:
+        description:
+          - The DB cluster (lowercase) identifier. The identifier must contain from C(1) to C(63) letters, numbers, or
+            hyphens and the first character must be a letter and may not end in a hyphen or contain consecutive hyphens.
+        aliases:
+          - cluster_id
+          - id
+        type: str
+        required: True
+    db_cluster_parameter_group_name:
+        description:
+          - The name of the DB cluster parameter group to associate with this DB cluster.
+            If this argument is omitted when creating a cluster, the default DB cluster parameter group for the specified DB engine and version is used.
+        type: str
+    db_subnet_group_name:
+        description:
+          - A DB subnet group to associate with this DB cluster if not using the default.
+        type: str
+    enable_cloudwatch_logs_exports:
```

Comment:
Do we need here a default list? Because `purge_cloudwatch_logs_exports` is default true?  I've no idea ...

Reply:
ah, nvm. it was `purge_...`